### DISCUSSION TITLE
Why can exported files not be imported?

### DISCUSSION THREAD

**Discussion by PEOM** [pawelsalawa/sqlitestudio#5587] [General]
### Details
I know there are other ways to make some of the changes in sqLite files I want, however, it would be a convenience to me if tables could be exported in csv format by SQLite Studio.

This application is very useful to me.

**Comment by pawelsalawa** [comment]
Sooo... this request is about importing files that were exported (as in the title) or about exporting table structures into CSV (as in the description)?
